packages/server/src/course/course.controller.ts

```diff
@@ -59,23 +57,16 @@ import { ConfigService } from '@nestjs/config';
 @Controller('courses')
 @UseInterceptors(ClassSerializerInterceptor)
 export class CourseController {
+  readonly MAX_ASYNC_QUESTIONS = 100;
+  readonly MAX_QUEUES_PER_COURSE = 30;
```

Comment:
Okay this might sound weird, but with some of the work i'm doing with demos and stuff, we're basically making it so each queue is an assignment (Ramon strongly recommend doing it like this), and 30 *might* be a little too low, or at least for some of the larger courses. Right now, I have it so it's recommended to have each queue be an assignment (so maybe upwards of 15 queues or something), but some professors may prefer to create a new queue for each lab section and assignment, leading to probably upwards of over 100 queues (e.g. 10 labs with 10 lab sections). Granted, when my feature comes, it would need to come with a way to easily hide/show/archive queues, but I thought maybe this is worth mentioning to maybe start a discussion about it.

Reply:
For performance reasons, having 100 queues seems like overkill. I don't like the idea that each assignment should have its own queue instead of repurposing it. I will chat with Ramon about that